apps/prairielearn/src/pages/administratorQueries/administratorQueries.ts

```diff
@@ -14,16 +15,21 @@ router.get(
   asyncHandler(async (req, res, _next) => {
     const fileList = await fsPromises.readdir(queriesDir);
     const jsonList = _.filter(fileList, (f) => /\.json$/.test(f));
-    res.locals.queries = await Promise.all(
+    const queries = await Promise.all(
       jsonList.map(async (f) => {
         const contents = await jsonLoad.readJSONAsync(path.join(queriesDir, f));
```

Comment:
I'd be inclined to define a Zod schema for the loaded JSON so that we're always validating that is has the expected properties and shape.

Reply:
Resolved in commit a1e7740.